project_name/utils/blocks.py

```diff
@@ -21,12 +21,18 @@ class Meta:
 
 
 class AccordionBlock(blocks.StructBlock):
-    heading = blocks.ListBlock(AccordionBlock())
-    list = blocks.ListBlock(AccordionBlock())
+    sections = blocks.ListBlock(
+        AccordionSectionBlock(),
+    )
 
     class Meta:
         icon = "list-ol"
-        template = "components/accordion/accordion.html"
+        template = "components/streamfield/blocks/accordion.html"
+
+    def get_context(self, value, parent_context=None):
+        ctx = super().get_context(value, parent_context=parent_context)
+        ctx["accordions"] = value["sections"]
+        return ctx
```

Comment:
It's probably simpler to remove the `get_context` override and use `{% for accordion in value.sections %}`, unless it's intentional to showcase the method override.

Reply:
it wasn't intentional on my part. I borrowed this from the charity kit. That does seem to be a more DRY approach and as far as I can tell, it didn't break anything. I'll go ahead and add that modification.